core/lib/dal/src/models/storage_transaction.rs

```diff
@@ -336,8 +337,14 @@ impl From<StorageApiTransaction> for api::Transaction {
 impl<'r> FromRow<'r, PgRow> for StorageApiTransaction {
     fn from_row(db_row: &'r PgRow) -> Result<Self, Error> {
         let row_signature: Option<Vec<u8>> = db_row.get("signature");
-        let signature = row_signature
-            .and_then(|signature| PackedEthSignature::deserialize_packed(&signature).ok());
+        let transaction_type = db_row
+            .try_get::<Option<i32>, &str>("tx_format")
```

Comment:
It's not for this PR. 
@perekopskiy should we mark tx_format as not null? seems it supposed to be always exist

Reply:
Agree, I would create a ticket to check if there are transactions with tx_format IS NULL, if not then add a constraint, otherwise investigate

Reply:
https://linear.app/matterlabs/issue/PLA-612/mark-tx-format-as-not-null-in-db